tests/test_items_view_model.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from todo_app.view_models.items_view_model import ItemsViewModel
+from todo_app.data.item import Item
+
+item_1 = Item(1, "Item 1", 1, "To Do")
```

Comment:
Why would we need 9 items in this test suite as opposed to 3 (one of each todo/doing/done)?